test/integration/rise-storage.html

```diff
@@ -354,25 +353,6 @@
           done();
         });
 
-        test("should retry storage request after rise-storage-api-error is encountered", function(done) {
```

Comment:
@rodrigopavezi The first test explicitly has the `_isCacheRunning` flag set to false. I'm assuming it's the same for the other test.